picard/mbxml.py

```diff
@@ -205,10 +205,23 @@ def artist_credit_to_metadata(node, m, release=False):
         m["~artists_sort"] = artistsort
 
 
+def country_list_from_node(node):
+    # Extracts iso_3166_1_codes for all countries in `release_event_list`
+    if "release_event_list" in node.children:
+        country = []
+        for re in node.release_event_list[0].release_event:
```

Comment:
It's potentially confusing when you clobber module names (`re` is imported into this file). Just use `release_event`.